### PR TITLE
Fix local docker compose setup

### PR DESCRIPTION
**Pull request by BMartinos**
## Summary by CodeRabbit

- **Chores**
	- Updated the path to the Dockerfile for the application container, improving the organization of project files.
	- Removed version declaration line from Docker Compose configuration, streamlining build process management.

### CODE REVIEW

**Comment by yassinedorbozgithub** [comment]
After the tests i can comfirm that this update fixes the docker up issues
image
image

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-
 services:
   app:
-    image: smart-health-links-portal:latest
     container_name: smart-health-links-portal
     build:
       context: .
       target: dev
-      dockerfile: Dockerfile
+      dockerfile: docker/development/Dockerfile
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docker-compose.yml:7]: I think for now this line stay as is for now as its pointing specifically to the development dockerfile. 

For production, there is a specific Dockerfile for building the production build image. Our CICD points to the production Dockerfile when building the new image
https://github.com/jembi/smart-health-links-portal/blob/main/docker/production/Dockerfile
https://github.com/jembi/smart-health-links-portal/blob/main/.github/workflows/docker-publish.yml#L36

**Comment by IkbelTalebHssan** [comment]
smart-health-links-portal instance is running successfully on port 3000.
LGTM :heavy_check_mark:

**Review by IkbelTalebHssan — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yassinedorbozgithub) [reply docker-compose.yml:7]: It make a full sense

**Review by yassinedorbozgithub — APPROVED** [review]

**Pull request merged**